### ISSUE TITLE
add resistors

### ISSUE THREAD

**Issue by hartytp** [sinara-hw/BNC_IDC#1]
@gkasprow since this board can be used as both an input to the ADC and as an output from the DAC, it might make sense to have a series 0R input resistor on each side of the capacitor.

Also, not sure if this makes sense or not, but we could add a DNF resistor in parallel with the capacitor in case anyone wanted to add an attenuator. It's a low priority, but might come in use. If you don't think it's worth it then don't bother :)

**Comment by gkasprow** [comment]
done
